CHANGELOG.md

```diff
@@ -19,9 +19,6 @@ We follow Semantic Versions.
 
 
 ## Version 0.1.2
-
```

Comment:
Probably an accident 🙂 
There's no need to edit existing CHANGELOG entries, only add new ones